languages/elixir/reference/exercise-concepts/secret-handshake.md

```diff
@@ -0,0 +1,53 @@
+# Concepts for secret-handshake
+
+## Optimal solution
+```elixir
+defmodule SecretHandshake do
+  use Bitwise
+
+  @doc """
+  Determine the actions of a secret handshake based on the binary
+  representation of the given `code`.
+
+  If the following bits are set, include the corresponding action in your list
+  of commands, in order from lowest to highest.
+
+  1 = wink
+  10 = double blink
+  100 = close your eyes
+  1000 = jump
+
+  10000 = Reverse the order of the operations in the secret handshake
+  """
+  @spec commands(code :: integer) :: list(String.t())
+  def commands(code) do
```

Comment:
The `do` bit defines a scope for the code below, so the "Scoping" Concept could be added too.

Reply:
That would be a really good one to add as variable scope is very strict in elixir due to its immutability rules.